rviz_msgs/UUID request_id
uint8 error_code
string error_string

uint32 value_doubled
